bundles/Billing/Controller/IndexController.php

```diff
@@ -21,7 +21,6 @@ class IndexController extends AbstractController
     /**
      * @Route("/show_export_invoice/{invoiceId}", name="api_show_export_invoice", methods={"GET"})
      *
-     * @param \Billing\Service\BillingService $billingService
      * @param $invoiceId
```

Comment:
Is this part of our new documentation strategy, i.e. to only “document” select parameters?

Reply:
This is just the result of applying cs-fixer where the new rule "no_superfluous_phpdoc_tags".